_posts/2022-09-26-changelog-179.markdown

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #179"
+date: 2022-09-26 13:00:00 +0300
+---
+
+This is the last release that supports the 2022.1 IDEs. To receive plugin updates further, please upgrade your IDE to 2022.2.
+
+## New Features
+
+* [#9241] Change the rendering of folded blocks for `use`, `mod`, `extern crate`, and `where` blocks to make them more distinct from folded block comments (by [@afetisov])
+
+  **TODO**: img/gif
+
+* [#9176] Use the [`--keep-going` Cargo option](https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#keep-going) when building and evaluating build scripts to continue building even if some crate fails to compile
```

Comment:
Let's add explanation how it affects users. Also, I'd mention that build script evaluation is disabled by default and can be turned on via experimental feature